client/src/main/java/com/linecorp/decaton/client/DecatonClient.java

```diff
@@ -79,6 +82,19 @@
      */
     CompletableFuture<PutTaskResult> put(String key, T task, Consumer<Throwable> errorCallback);
 
+    /**
+     * Put a task onto a specified kafka partition.
+     * Note that the specified partition here is prioritised over the result of
+     * {@link Partitioner#partition(String, Object, byte[], Object, byte[], Cluster)}.
```

Comment:
I think we do not need to mention about precedence here explicitly, because this is KafkaProducer's behavior rather than DecatonClient's interface.